ujson/src/ujson/IndexedValue.scala

```diff
@@ -10,12 +10,12 @@ import upickle.core.{Visitor, ObjVisitor, ArrVisitor, Abort, AbortException}
   * want to work with an AST but still provide source-index error positions if
   * something goes wrong
   */
-@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead")
+@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead", "After 3.1.4")
```

Comment:
From experience, and as suggested in the Scala documentation, we want the name of the library in the version field. Otherwise, the compiler message is hard to get.

```suggestion
@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead", "upickle after 3.1.4")
```
or 

```suggestion
@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead", "ujson after 3.1.4")
```

Reply:
done